torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -53,21 +53,50 @@ static IValue Table(
   return Tup(std::move(ivalue_entries));
 }
 
-std::string getModulePath(Node* node) {
-  std::string modulePath = node->scopeName();
-  size_t end = modulePath.size();
-  // Here we remove the source range information to make the
-  // module debugging information shorter and cleaner.
-  if (modulePath[end - 1] == '>') {
-    end = modulePath.rfind('<');
-    if (end > 0 && modulePath[end - 1] == '<') {
-      --end;
+std::string getModulePath(Node* node, const std::string& root_scope_string) {
+  if (!node->callstack()) {
+    return root_scope_string + ".forward";
+  } else {
+    std::string module_info = root_scope_string;
+    auto callstack_ptr = *(node->callstack());
+    const auto& vec = callstack_ptr->vec();
+
+    for (const auto& element : vec) {
+      const auto& tup = element;
+      const auto& opt_module_instance_info = std::get<2>(tup);
```

Comment:
Can you create constants for element numbers such as 0, 1 and 2. e.g. 
```
constexpr size_t kFunction = 0;
auto function_name = std::get<kFunction>(element)->name;
```
Might make code more readable.